code/modules/recycling/disposal/pipe.dm

```diff
@@ -95,9 +95,11 @@
 
 	if(isfloorturf(T)) //intact floor, pop the tile
 		floorturf = T
-		if(floorturf.floor_tile)
+		if(floorturf.forced_plating == TRUE)
+			floorturf.pop_tile = TRUE
+		if(floorturf.floor_tile && floorturf.pop_tile == TRUE)
 			new floorturf.floor_tile(T)
```

Comment:
no, because this is only occuring *in this instance* ... i dont want to override the tile directly because this will not apply in any and all instances, *only* in this instance (a disposals pipe busting through the floor) will this ever occur, so overriding it on the stuff directly will mess with normal functioning behaviour